### PR TITLE
Tokens issued with incorrect scopes for Access Points (SOFTWARE-4772)

### PR DESCRIPTION
**Pull request by brianhlin**
Adding the server side checks to make sure that AP vs EE token requests have the correct scopes. Still need to work on updating the client but figured that this was enough for an initial review.

### CODE REVIEW

File: registry/token/views.py [hunk 1]
```diff
@@ -24,7 +24,9 @@ def code_get():
 
 SOURCE_PREFIX = "RESOURCE-"
 SOURCE_POSTFIX = "flock.opensciencegrid.org"
-ALLOWED_AUTHORIZATIONS = {"READ", "ADVERTISE_STARTD", "ADVERTISE_MASTER"}
+BASE_ALLOWED_AUTHORIZATIONS = {"READ", "ADVERTISE_MASTER"}
+AP_ALLOWED_AUTHORIZATIONS = {"ADVERTISE_SCHEDD", "DAEMON"}
```
Comment (bbockelm) [inline registry/token/views.py]: Can we leave out `DAEMON` for now?

I don't think it's required even if it shuts up complaints in the logs.

Comment (brianhlin) [reply registry/token/views.py]: Done

**Review by bbockelm — APPROVED** [review]
Ok, I think we're good to go.

Obviously I have not tested the functionality but the concepts look correct.

**Comment by brianhlin** [comment]
Let's see if it works on the ITB! Client bits to come in a separate PR

**Pull request merged**